examples/pulse_control/2_Native_gate_calibrations.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>1) "pulses that <strong>are</strong> carefully"</p><p> </p><p>2) Modify it -&gt; what is it referring to here?</p><p> </p><p>3) Obtaining <s>the</s> accurate sequences <s>that are</s></p><p> </p><p>4) "First, it..." -&gt; this makes it sound like there is a second. Where is the second being mentioned? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>